src/frontend/src/selenium.test.ts

```diff
@@ -533,8 +533,8 @@ test("Register new identity and add additional device", async () => {
     await driver.get(II_URL);
     const userNumber = await registerNewIdentity(driver);
     await on_Main(DEVICE_NAME1, driver);
+    await removeVirtualAuthenticator(driver, firstAuthenticator);
```

Comment:
Maybe add a comment explaining that the first authenticator is removed as a workaround to flakyness, since this is not really expected / obvious?